src/pages/developers/learning-tools.js

```diff
@@ -80,6 +80,15 @@ const LearningToolsPage = ({ data }) => {
   const intl = useIntl()
 
   const sandboxes = [
+    {
+      name: "Replit",
+      description: "page-learning-tools-replit-description",
+      url: "https://replit.com/@replit/Solidity-starter-beta",
+      image: getImage(data.replit),
+      alt: "page-learning-tools-replit-logo-alt",
+      background: "#11131D",
+      subjects: ["Solidity", "web3"],
+    },
```

Comment:
@sorenrood can you double check that this is correct?

Reply:
Link looks good. Can we use `#0f1524`?

Reply:
done.